build/single.html

```diff
@@ -6,21 +6,63 @@
     <link rel="stylesheet" type="text/css" href="style.css" />
     <link rel="stylesheet" type="text/css" href="styles/responsive.css" />
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,700" rel="stylesheet" />
+    <link rel="stylesheet" href="http://i.icomoon.io/public/48159919ac/test/style.css"></link>
+    <link rel="stylesheet" type="text/css" href="fonts/icomoon/icomoon.css" />
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
     <script type="text/javascript" src="scripts/interaction.js"></script>
 </head>
 <body>
+    <div class="banner">
+
+    </div>
     <div class="header">
         <div class="wrapper">
             <a class="logo" href="#">
                 <img src="img/logo.png" alt="" />
             </a>
+            <h2 class="trouble">Troubleshooting</h2>
         </div>
     </div>
 
     <div class="article full section content">
-        <div class="wrapper">
-            <p>Lorem ipsum dolor sit amet.</p>
+        <div class="main wrapper">
+          <h2>Network Roaming Configuration</h2>
+          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Officiis omnis reiciendis non aut nesciunt, illum distinctio voluptatibus, placeat iste soluta adipisci beatae ipsam, quod sapiente blanditiis doloremque aspernatur deserunt voluptate! Laborum odit ipsum repudiandae! Libero aperiam ducimus dolorem hic quos quia quam omnis eos facilis, sequi, ut distinctio assumenda, accusantium.</p>
+
+          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt quaerat, tempore omnis quasi quam, dignissimos voluptatum ullam architecto accusantium facere sit non facilis consequuntur sint sed at, neque tempora quibusdam asperiores quisquam molestiae necessitatibus ipsum ea quia. Veniam eius non cumque eaque enim nesciunt, vel repellendus eum debitis magnam! Culpa odit quaerat dignissimos consequuntur omnis molestias rerum ut praesentium. Animi possimus maxime, minus laborum atque doloribus officiis magni qui non perferendis, neque tempore facilis iusto a voluptate hic aliquid deleniti ad explicabo aut quas eveniet asperiores nulla tenetur minima. Architecto delectus dolore quasi natus reprehenderit explicabo, magni totam, nobis tenetur.</p>
+
+          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsum corporis officiis, nam nostrum ipsa beatae reprehenderit aut dolor commodi dolorem consectetur optio repellendus, facere ratione minus delectus veniam placeat. Commodi excepturi, animi, fuga optio reiciendis explicabo soluta nihil nisi. Modi repudiandae officia consequatur in, laborum labore explicabo inventore quidem voluptatum eligendi architecto nihil omnis maiores animi vero, veniam excepturi, necessitatibus, totam obcaecati odio dolorum dolor error aliquam et facilis. Quibusdam ut, doloremque in cupiditate voluptate!</p>
+        </div>
+        <div class="side-bar wrapper">
+          <div class="table">
```

Comment:
You only want to use `<table>` elements when you're actually showing tabular data (like how a typical excel file looks). An unordered list (`<ul>`) is better suited for this, or even just `<a>` tags within a `<div>`.

This would be how I would have structured the html:
```
        <div class="side-bar">
            <div class="mod">
                <h2>More Articles</h2>
                <ul>
                    <li><a href="#">Lorem ipsum</a></li>
                    <li><a href="#">Lorem ipsum</a></li>
                    <li><a href="#">Lorem ipsum</a></li>
                    <li><a href="#">Lorem ipsum</a></li>
                    <li><a href="#">Lorem ipsum</a></li>
                </ul>
            </div>
        </div>
```

Reply:
This would be my CSS for the `side-bar`: ```.side-bar {
	flex: 1 1 auto;
}

.side-bar .mod {
	border: 1px solid #ddd;
}

.side-bar h2 {
	color: #333;
	padding: 16px;
	font-weight: 600;
	line-height: 1em;
	background: #fafafa;
	font-size: 18px;
	margin: 0;
	border-bottom: 1px solid #ddd;
}

.side-bar ul {
	margin: 0;
	padding: 0;
}

.side-bar li {
	margin: 0;
	padding: 0;
	border-bottom: 1px solid #ddd;
	list-style: none;
}

.side-bar li:last-child {
	border-bottom: 0;
}

.side-bar a {
	padding: 12px 16px;
	display: block;
	text-decoration: none;
	color: #333;
	font-weight: 400;
}

.side-bar a:hover {
	text-decoration: underline;
	background: #fafafa;
	color: #cc0000;
}```

Reply:
I just threw a lot at you there. Feel free to respond to these comments, or ask on slack if you have questions on any of that

Reply:
I absolutely understand all of the above, which is great. But I just need to start thinking differently. For example, I wouldn't have thought to add the hover rule.